### DISCUSSION TITLE
CXR module: Text report vs CSV file for presence of pneumonia

### DISCUSSION THREAD

**Discussion by shrutiihegde** [MIT-LCP/mimic-code#1523] [MIMIC-IV-CXR]
I wanted to bring to your attention an observation I made while reviewing the MIMIC-CXR Database for my study. I noticed that in some cases, the text report indicated no finding of pneumonia, but the corresponding labels in the CSV file indicated the presence of pneumonia.

For instance, in the case of p12288954 : s53632981, the text report showed no finding of pneumonia, but the CSV label indicated a presence of pneumonia with a value of 1. Similarly, in the case of p12572662 : s52904827, a discrepancy was observed.

I'm curious if there may be more to this issue than what I currently understand, and I would greatly appreciate some clarity on this matter. Thank you for your time.

**Comment by alistairewj** [comment]
Nope, the labeling is definitely not perfect. It was done automatically by either the NegBio or the CheXpert tool. If you want to re-run the labeler yourself the code to do that is here: https://github.com/MIT-LCP/mimic-code/tree/main/mimic-iv-cxr/txt
